community/modules/scripts/htcondor-install/files/autoscaler.py

```diff
@@ -295,22 +296,22 @@ def scale(self):
 
                     if server not in node_busy:
                         if status == "Unclaimed":
-                            node_busy[server] = False
+                            node_busy[server] = zone
```

Comment:
The `node_busy` map is not well named anymore. You could consider renaming it to something like `server_zone` or setting it to a tuple or map of (busy:bool, zone:string).

Reply:
Resolved on HEAD of PR. Called it `idle_node_zones`. This code is in serious need of re-implementation (shell call-outs are unnecessary as HTCondor has Python interface).